From e326198b4e9437b5480d52fb9fb6d3a635421111 Mon Sep 17 00:00:00 2001
From: Hugues Fruchet <hugues.fruchet@st.com>
Date: Thu, 12 Jan 2017 16:57:27 +0100
Subject: [PATCH 24/25] v4l2object: allows a single OUTPUT buffer

Patch needed waiting for V4L2 request API implementation
in DELTA driver.

Change-Id: I8399119a544854da4f01000706f68c6a0cbc4a6c
Reviewed-on: https://gerrit.st.com/63441
Reviewed-by: Hugues FRUCHET <hugues.fruchet@st.com>
Tested-by: Hugues FRUCHET <hugues.fruchet@st.com>
---
 sys/v4l2/gstv4l2object.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sys/v4l2/gstv4l2object.h b/sys/v4l2/gstv4l2object.h
index 4c33e96..d706f90 100644
--- a/sys/v4l2/gstv4l2object.h
+++ b/sys/v4l2/gstv4l2object.h
@@ -39,7 +39,7 @@ typedef struct _GstV4l2Xv GstV4l2Xv;
 #include <gstv4l2bufferpool.h>
 
 /* size of v4l2 buffer pool in streaming case */
-#define GST_V4L2_MIN_BUFFERS 2
+#define GST_V4L2_MIN_BUFFERS 1
 
 /* max frame width/height */
 #define GST_V4L2_MAX_SIZE (1<<15) /* 2^15 == 32768 */
-- 
2.7.4

